src/derive_utils.rs

```diff
@@ -312,14 +321,8 @@ impl ModuleDef {
         #[cfg(not(Py_3_7))]
         ffi::PyEval_InitThreads();
```

Comment:
Missing a `unsafe` block here.

Reply:
Actually I don't think this call to `PyEval_InitThreads` is needed at all. Pushed a commit to remove it, and updated the OP.